packages/presence/src/server.ts

```diff
@@ -60,6 +60,13 @@ export default class Server implements Party.Server {
 			this.updateUsers()
 		}
 	}
+
+	onRequest(req: Party.Request): Response | Promise<Response> {
+		if (req.method === 'POST') {
+			return Response.json(this.getPresenceMessage().payload)
+		}
```

Comment:
Can we do a little bit of routing here so it can have a more descriptive path?

Reply:
yes we can! But I discovered a bug, that we need to land this fix for https://github.com/partykit/partykit/pull/503 Might be a day before that happens.

Reply:
actually, I have a workaround that doesn't need us to land that fix. gimme a min.

Reply:
aight, whaddaya think?